### PR TITLE
Add Credit Card as a PII field

### LINKED ISSUE #42: Allow uploading PII records with "credit_card" fingerprints.

### LINKED ISSUE #50: Update libfp to 2019-02 Release
Roll in the new version of libfp that changes the output email fingerprint format.

### PR DESCRIPTION
**Pull request by mattgeiger**
This adds 'credit_card' as a parameter to `add_pii()`. This field will be fingerprinted & uploaded to the API.

**Note: ** The API receiving the request must be updated to accept this data.

Closes #42 
Closes #50

### CODE REVIEW

File: src/matchlight/record.py [hunk 1]
```diff
@@ -245,6 +247,10 @@ def add_pii(self, project, description, email, first_name=None,
             phone_fingerprints = fingerprints_pii_phone_number(phone)
             data['phone_fingerprints'] = [phone_fingerprints]
 
+        if credit_card is not None:
```
Comment (cmeadows) [inline src/matchlight/record.py]: Should this rather be `if not credit_card`?

Comment (mattgeiger) [reply src/matchlight/record.py]: Do you mean `if credit_card`?

Comment (cmeadows) [reply src/matchlight/record.py]: Yeah. The `None` was negating, so I typed really fast.

Comment (mattgeiger) [reply src/matchlight/record.py]: I think that's a worthwhile update. I don't see the utility in fingerprinting empty strings.

**Comment by acviana** [comment]
Would this MR also close this issue: https://github.com/TerbiumLabs/python-matchlightsdk/issues/50?

**Comment by mattgeiger** [comment]
> Would this MR also close this issue: #50?

Yes, I forgot about that issue. I have updated this PRs description.

**Comment by acviana** [comment]
This looks good, Thanks @mattgeiger!!

I want to hold off on merging this until after we get #57 merged because I want to release off that merge.

I want to release this next but want to pull in a few other changes as well that I'll add to a milestone.

**Comment by mattgeiger** [comment]
@acviana in reference to this:
> I want to hold off on merging this until after we get #57 merged because I want to release off that merge.

Since the target for this MR is now "development",  can I go ahead & merge this?

**Comment by acviana** [comment]
@mattgeiger, yes thank you.

**Pull request merged**